plutus-tx/src/PlutusTx/Lift.hs

```diff
@@ -57,12 +59,15 @@ safeLift
        , PLC.AsFreeVariableError e
        , AsError e uni fun (Provenance ()), MonadError e m, MonadQuote m
        , PLC.Typecheckable uni fun
+       , PrettyPrintable uni fun
        )
     => a -> m (UPLC.Term UPLC.NamedDeBruijn uni fun ())
 safeLift x = do
     lifted <- liftQuote $ runDefT () $ Lift.lift x
     tcConfig <- PLC.getDefTypeCheckConfig $ Original ()
-    compiled <- flip runReaderT (toDefaultCompilationCtx tcConfig) $ compileTerm lifted
+    -- NOTE:  Disabling simplifier, as it takes a lot of time during runtime
```

Comment:
Is this true? Or is this only true when we inline types? I think we should turn this back on again, but I want to merge this so let's do it later.